docs/source/migration_guides.md

```diff
@@ -166,6 +166,18 @@ The version of backbone that ipywidgets depend on has changed from 1.2.3 to 1.4.
 
 Note: If you were relying on setting certain instance attributes via the `extend` method, you might now need override the `preinitialize` method in order for their values to be set in time.
 
+#### Custom tag names
+
```

Comment:
ipydatawidgets does this typescript magic using `// @ts-ignore: 2611`

Reply:
```
/**
  * The default tag name.
  *
  * #### Notes
  * This is a read-only attribute.
  */
 // @ts-ignore: 2611
 get tagName() {
   // We can't make this an attribute with a default value
   // since it would be set after it is needed in the
   // constructor.
   return 'img';
 }
```

Reply:
Having looked at this more closely, these are the points that I think should be captured in the docs:
- If anyone is overriding one of these attributes on a View, these docs apply: `['model', 'collection', 'el', 'id', 'attributes', 'className', 'tagName', 'events']`. These attributes are used by backbone in the constructor before calling `initialize`.
- In new version of backbone, there is a `preinitialize` method which can be used to override the values.
- In the old version of backbone, you had two options (which are still valid):
  - Override them with a property, such that the getter will always shadow access attempts. Typescript now complains about this, but that can be ignored with `// @ts-ignore: 2611`.
  - Override the constructor, and pass these options to the super constructor (https://backbonejs.org/docs/backbone.html#section-167).

I'll see if I can make an updated text

Reply:
Hmm. Maybe it is actually only relevant for `tagName`, as it is the only one with a default value set: https://backbonejs.org/docs/backbone.html#section-171